packages/wallet-ui/src/components/ui/organism/Header/SendModal/SendModal.view.tsx

```diff
@@ -108,8 +180,9 @@ export const SendModalView = ({ closeModal }: Props) => {
             </Network>
             <AddressInput
               label="To"
-              placeholder="Paste recipient address here"
+              placeholder="Paste recipient address or .stark name here"
               onChange={(value) => handleChange('address', value.target.value)}
+              resolvedAddress={resolvedAddress}
```

Comment:
AddressInput is shared with other component, and i saw it has added some logic for checking the starkname internally, so it will mix up the use case

IMO, 
either 
- we build a new AddressInput for that case
or, 
- we rollback and takeout the validate starkname logic from AddressInput, 
- we pass an flag to stop validate anything inside the AddressInput
- we do the validation at parent level, and display the error from parent